webapp/components/Comment.vue

```diff
@@ -23,10 +23,27 @@
     <!-- eslint-disable vue/no-v-html -->
     <!-- TODO: replace editor content with tiptap render view -->
     <ds-space margin-bottom="small" />
-    <div style="padding-left: 40px;" v-html="comment.contentExcerpt" />
-    <!-- eslint-enable vue/no-v-html -->
+    <div v-if="!isActive" v-html="comment.contentExcerpt" style="padding-left: 40px;" />
+    <div v-show="comment.content !== comment.contentExcerpt">
+      <span
+        v-if="!isActive"
```

Comment:
Call the property `isCollapsed`. From the name `active` would expect only one comment to be active at a time.But `isActive` refers only to sth. visible here.

Reply:
Done